research-hub-web/src/app/components/subhubs/subhubs.component.ts

```diff
@@ -0,0 +1,120 @@
+import { Component, OnInit } from "@angular/core";
+import { ActivatedRoute } from "@angular/router";
+import {
+  AllSubHubChildPagesGQL,
+  AllSubHubChildPagesQuery,
+  AllContentItemParentSubHubsGQL,
+  AllContentItemParentSubHubsQuery,
+} from "../../graphql/schema";
+import { Observable } from "rxjs";
+import { pluck } from "rxjs/operators";
+
+@Component({
+  selector: "app-subhubs",
+  templateUrl: "./subhubs.component.html",
+  styleUrls: ["./subhubs.component.scss"],
+})
+export class SubhubsComponent implements OnInit {
+  public allSubHubChildPages$: Observable<
+    AllSubHubChildPagesQuery["subHubCollection"]
+  >;
+
+  public allContentItemParentSubHubs$: Observable<
+    AllContentItemParentSubHubsQuery["subHubCollection"]
+  >;
+
+  public parentSubHubs =  [];
+
+  constructor(
+    private route: ActivatedRoute,
+    public AllSubHubChildPagesGQL: AllSubHubChildPagesGQL,
+    public AllContentItemParentSubHubsGQL: AllContentItemParentSubHubsGQL
+  ) {}
+
+  ngOnInit(): void {
+    this.route.params.subscribe((params) => {
+      const currentPageSlug = params["slug"];
+      // test slug: landing-page-for-a-sub-hub
+
+      if (!!currentPageSlug) {
+        console.log(currentPageSlug);
```

Comment:
Console log that can be removed.

Reply:
Removed (hope you don't mind @warrickct ) ✌️